.github/workflows/tests-js.yml

```diff
@@ -0,0 +1,28 @@
+name: 'JS Tests'
+on:
+  pull_request:
+    paths:
+      - 'resources/js/**'
+      - 'package.json'
+      - 'webpack.*.js'
+      - 'jest.config.json'
+      - 'jest.setup.js'
+      - 'jsconfig.json'
+      - '.github/workflows/tests-js.yml'
```

Comment:
2 notes:
1. `npm run test` runs in the `src/modules` folder, for block editor tests.
2. curious why we wouldn't run these tests anytime we have a PR?

Reply:
Updated to include that path.

The idea is that if we aren't touching anything the JS or the JS tests rely on, it's not worth running/waiting on the action itself.

Reply:
If we change any code in common, we don't change any JS code, but can affect tests.